solutions/mint-nft/README.md

```diff
@@ -1,10 +1,10 @@
 # mint-nft example
 
-This example shows how -------------------------
+This example shows how to mint an NFT using Next.js and [Moralis](https://moralis.io/)
 
 ## Demo
 
-https://mint-nft.vercel.app
+https://mint-nft-kappa.vercel.app/
```

Comment:
We need a better example name, e.g: `solutions-mint-nft`, and also remove the trailing slash.

Reply:
Good point! I claimes solutions-mint-nft and removed the slash 👍